sonar/modules/shibboleth_authenticator/utils.py

```diff
@@ -49,7 +50,10 @@ def get_account_info(attributes, remote_app):
     return dict(
         user=dict(
             email=email,
-            profile=dict(full_name=full_name, username=slugify(full_name)),
+            profile=dict(full_name=full_name,
```

Comment:
Can you put some comments here to explain the utility of each returned property?